testsuite/tests/typing-objects/errors.ml

```diff
@@ -40,7 +40,8 @@ let foo = 42#m;;
 Line 1, characters 10-12:
 1 | let foo = 42#m;;
               ^^
-Error: This expression is not an object; it has type "int"
+Error: This expression is not an object;
+              it has type "int"
```

Comment:
I wonder why there is this large left-indentation here.

Reply:
A missing `\` before a newline.